app/models/govbox/api_connection_with_obo_support.rb

```diff
@@ -26,8 +26,5 @@ def destroy_with_box?
 
   def validate_box(box)
     box.errors.add(:obo, :not_allowed) if obo.present?
-    if tenant
-      box.errors.add(:settings_obo, :blank) if box.settings_obo.blank?
-    end
```

Comment:
Obo je uz volitelne pri Govbox::ApiConnectionWithOboSupport.

Reply:
A ktory connection je teda pre "nas" certifikat a obo (ktore vieme zadat len my) ?

Reply:
`Govbox::ApiConnection`, pri tom sa nemeni nic, to uz je aktualny stav